utility/unit_of_work/module.go

```diff
@@ -1,12 +1,10 @@
 package unit_of_work
 
 import (
-	"fmt"
-
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
-	"github.com/pokt-network/pocket/shared/mempool"
 	"github.com/pokt-network/pocket/shared/modules"
 	"github.com/pokt-network/pocket/shared/modules/base_modules"
+	"go.uber.org/multierr"
```

Comment:
@dylanlott  Let's change all of this in a small followup commit

Reply:
Covered in #939